setup.py

```diff
@@ -57,7 +57,7 @@ def run(self):
 
 
 setup(name='citrine',
-      version='0.9.1',
+      version='0.9.2',
```

Comment:
Version is out of date

Reply:
I must have missed something, seems like a large bump to go to .14?

Reply:
You missed us starting to enforce in an automated way that all commits bump the version, plus a few separate commits that added new methods

Reply:
Ok, cool. I did actually bump the version initially too, but that must have gotten stale.